src/api/app/controllers/person/token_controller.rb

```diff
@@ -14,7 +14,12 @@ def index
     def create
       authorize @user, :update?
 
-      pkg = (Package.get_by_project_and_name(params[:project], params[:package]) if params[:project] || params[:package])
+      unless params[:project] && params[:package]
+        render_error status: 400, errorcode: 'missing_parameters', message: 'The package and project parameters must be provided.'
```

Comment:
Hey @mfriedrichgh we have a exception class for this :) so you can use it here https://github.com/openSUSE/open-build-service/blob/ee8f92cfc8134dafe3d541bec26a72588ccd9a91/src/api/lib/api_error.rb#L84